libs/utils/src/ballot_package.ts

```diff
@@ -133,10 +133,11 @@ async function readJsonEntry(zipfile: ZipFile, entry: Entry): Promise<unknown> {
 }
 
 async function readBallotPackageFromZip(
```

Comment:
should the name change to `FromBuffer` or `FromZipBuffer`?

Reply:
> should the name change to `FromBuffer` or `FromZipBuffer`?

Probably better, yes. Will do another PR (guess I shouldn't use auto-merge 😄).